src/RoverSelection/RoverSelection.tsx

```diff
@@ -1,6 +1,25 @@
+import { RoverSummary } from "./RoverSummary/RoverSummary";
+import roverData from "./roverData.json";
+import { Link } from 'react-router-dom';
+
 function RoverSelection() {
-    return (
-      <div>Rover selection page</div>
+
+  return (
+     <div>
+      <h1> Rover selection page </h1>
+        <div className='CuriosityRover'>   
+        <RoverSummary rover={roverData.curiosity} />
```

Comment:
Think we can format this file with shift + alt + f to fix identation

Reply:
sorry i feel like you spend your life telling me to do this haha, done!!

Reply:
I just love pretty code 😂